bin/update-ips.bash

```diff
@@ -352,7 +352,10 @@ allow_host() {
   local config_option="${2}"
   local host="${3}"
 
-  if is_ip_address "${host}"; then
+  if [[ "${host}" =~ \.cluster\.local$ ]]; then
+    log_warning "Allowing all for cluster local endpoint ${host}"
+    yq_eval "${config_file}" "${config_option}" "${config_option} = [ \"0.0.0.0/0\" ]"
```

Comment:
I'm ok with this as a temporary fix.
But this will always be a service ip, right? Can we (in some easy way) get the service CIDR and use that instead? I did a quick check but didn't see that in the config. But it should be possible to get from the kubeadm configmap in the cluster (if we can trust that that is always there) at `data.ClusterConfiguration.networking.serviceSubnet`.
What do you think about trying to add that subnet instead?

Reply:
I'll test it out.